pkg/controller/kataconfig/openshift_reconciler.go

```diff
@@ -414,8 +418,8 @@ func (r *ReconcileKataConfigOpenShift) listKataPods() error {
 func (r *ReconcileKataConfigOpenShift) kataOcExists() (bool, error) {
 	kataOcMcp := &mcfgv1.MachineConfigPool{}
 	err := r.client.Get(context.TODO(), types.NamespacedName{Name: "kata-oc"}, kataOcMcp)
-	if err != nil && errors.IsNotFound(err) {
-		r.kataLogger.Error(err, "No kata-oc machine config pool found!")
+	if err != nil && apimachineryErrors.IsNotFound(err) {
+		r.kataLogger.Info("No kata-oc machine config pool found!")
```

Comment:
It's not an error? We moved from Error to Info, so just checking.

Reply:
I changed it to Info on puropose. We don't expect it to exist, we're just checking but both is fine. NotFound is the good case of all error reasons. Thanks!